src/main/java/com/synopsys/integration/alert/web/config/JobConfigActions.java

```diff
@@ -150,7 +149,7 @@ public JobFieldModel updateJob(final UUID id, final JobFieldModel jobFieldModel)
         final Set<FieldModel> jobFieldModels = jobFieldModel.getFieldModels();
         for (final FieldModel fieldModel : jobFieldModels) {
             final FieldModel beforeUpdateEventFieldModel = fieldModelProcessor.performBeforeUpdateAction(fieldModel);
-            final Long fieldModelId = contentConverter.getLongValue(beforeUpdateEventFieldModel.getId());
+            final Long fieldModelId = Long.parseLong(beforeUpdateEventFieldModel.getId());
```

Comment:
Does this method differ in null handling than the one it is replacing?

Reply:
This will throw a NumberFormatException if the Id is null or the empty string.  We would return null in the past if the id was null or the empty string.

Reply:
In the past we used contentConverter which does what Paulo stated. But these are ID's that are all generated and maintained by us. At no point should we have our UI hit this endpoint without ID's specified. With the nul, we would fail silently if we messed something up in the UI. Now, we'll get NumberFormatException's if we mess it up. Perhaps there is a better way to do this, but I felt this was a better approach to the issue at the moment.